testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/AbstractHttpServerTest.groovy

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.test.base
+
+import ch.qos.logback.classic.Level
+import io.opentelemetry.instrumentation.test.AgentTestRunner
+import io.opentelemetry.instrumentation.test.utils.OkHttpUtils
+import io.opentelemetry.instrumentation.test.utils.PortUtils
+import okhttp3.OkHttpClient
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import spock.lang.Shared
+
+abstract class AbstractHttpServerTest<SERVER> extends AgentTestRunner {
+  public static final Logger SERVER_LOGGER = LoggerFactory.getLogger("http-server")
+  static {
+    ((Logger) SERVER_LOGGER).setLevel(Level.DEBUG)
```

Comment:
It used to be `ch.qos.logback.classic.Logger` previously

Also, why not set it in the logback XML config instead?

Reply:
Thanks, I have restored the cast to `ch.qos.logback.classic.Logger`. I'll see if I can move the configuration to xml.

Reply:
The same pattern is also used in AgentTestRunner and OkHttpUtils. @mateuszrzeszutek If it is ok with you I would leave this as is to avoid putting too much stuff into this pr. Additionally I'm not sure how well it would work out, there are some tests that also include logback.xml. I suspect that having a logback.xml in testing-common could override logback.xml that is inside test.

Reply:
Sure, we can tackle this issue some time later